sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1305,6 +1312,10 @@ export class Users {
 }
 
 
+// Warnings were encountered during analysis:
+//
+// src/client/Container/ContainerDefinition.ts:23:5 - (ae-forgotten-export) The symbol "GeospatialType" needs to be exported by the entry point index.d.ts
```

Comment:
and this warning?

Reply:
It is fine. I have missing exports elsewhere